app/assets/stylesheets/accounts.scss

```diff
@@ -34,6 +34,7 @@
     text-align: center;
     position: relative;
     z-index: 2;
+    text-shadow: 0 0 2px #000;
```

Comment:
This is not really a color that should change along with the rest of the black in the application.